build.gradle

```diff
@@ -343,7 +342,7 @@ def getGitTag() {
   return tag.name
 }
 
-def getAnnotatedTags() {
+ def getAnnotatedTags() {
```

Comment:
This is probably a typo?

Reply:
sorry, right. The IDE proposed to use "static" , I tried that but it broke things. After that I didn't clean up properly.